source/manual/icinga.html.md

```diff
@@ -0,0 +1,62 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Icinga
+section: Monitoring
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-08-16
+review_in: 6 months
+---
+
+* [Production](https://alert.publishing.service.gov.uk)
+* [Staging](https://alert.staging.publishing.service.gov.uk)
+* [Integration](https://alert.integration.publishing.service.gov.uk)
+* [AWS Production](https://alert.blue.production.govuk.digital)
```

Comment:
I'd be tempted to group the production ones together at the top to imply their both as important as each other.